### PR TITLE
Bug 1823297: Change default bus to SCSI for CD-ROM drives added from details tab

### PR DESCRIPTION
**Pull request by pcbailey**
This PR changes the default bus used for CD-ROM drives added from the VM details tab from VIRTIO to SCSI.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const getCDsPatch = (
       const disk: CD = {
         name,
         bootOrder: existingCD ? bootOrder : newBootOrder,
-        cdrom: { bus: bus || 'virtio' },
+        cdrom: { bus: bus || 'scsi' },
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts:63]: can you use `DiskBus.SCSI.getValue()` instead?

Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts]: Sorry. I didn't know we had an enum for that. That's much better, of course. =)

**Pull request merged**